views/template/demo.pug

```diff
@@ -0,0 +1,8 @@
+extends layout.pug
+
+block head
+	center
```

Comment:
Won't use center tags.
Sidebar right? (Although the block can be called navigation)
Yes, I'm aware of that, just wanted to set up the pages first. Will add it in the next push.